pages/developers/guidelines.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: page
-title: Package requirements and guidelines
+title: New package guidelines
```

Comment:
See comment below - these really are "package requirements and guidelines"? What do you not like about "requirements"?

Reply:
`New package guidelines` or `New package requirements` or `package guidelines` or `packages requirements` are all file, I just don't like this wrapping over to two lines and making the side bar ugly, and drawing too much attention to itself. This is really meant to be a short, menu title. It's probably most common that we will point a new user at this page directly, actually.

Reply:
Then there's no need to add "new" because these are simply the org guidelines. For a short title (I see the reason, a good one) just go with "Package requirements" then, to still make the point that we have some reasonable rules to try and have some sense of unity ...